content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -347,11 +347,11 @@
 			]
```

Comment:
It's not really related but we miss one endpoint here: PATCH several product models. Can you add it in this PR?